### ISSUE TITLE
Documentation sites out of date

### ISSUE THREAD

**Issue by jwodder** [PyGithub/PyGithub#541]
The PyGithub documentation (both <http://pygithub.readthedocs.io/en/latest/> and <https://pygithub.github.io/PyGithub/v1/index.html>) is still at version 1.25.2 (released two and a half years ago), even though the project is now at version 1.32.  Please fix this.

**Comment by NicoHood** [comment]
Please!

**Comment by ndpete** [comment]
So I don't know much about Sphinx but I tried to build the docs locally and this is a little of what I found. Hopefully this helps someone with a little more knowledge than me.

Line 324 in doc/conf.py will cause the build to fail immediately. To troubleshoot I commented it out.
Line 356 in doc/conf.py will also fail so I commented that out. 

After that It appears the documentation builds successfully however looking through it it looks like some of the newer undocumented methods I've used are formatted funny. And the changelog stuff is super screwy.

The conf.py file I was able to get a passing build with: [My Fork](https://github.com/ndpete/PyGithub/blob/master/doc/conf.py)
